rpm/generic/zfs.spec.in

```diff
@@ -70,6 +66,19 @@
 %define _systemd 1
 %endif
 
+# When not specified default to distribution provided version.  This
+# is normally Python 3, but for RHEL <= 7 only Python 2 is provided
+# from default repositories.
+%if %{undefined _python_version}
+%if 0%{?rhel} <= 7
```

Comment:
This triggers on Fedora as well. You want `0%{?rhel} && 0%{?rhel} <= 7`.

Reply:
Thanks, fixed.